### ISSUE TITLE
Условия vs Код

### ISSUE THREAD

**Issue by hlogeon** [camonroad/ico#2]
Прочешите внимательно еще раз все условия которые Вы пишите текстом(будь то whitepaper или readme на гитхаб) и убедитесь, что они соответствуют коду. Иначе это может вызвать разгого рода проблемы, от юридических до репутационных.
Вот например в readme вы пишите:

> если ICO не достигнет минимальной цели (не будет собрано 10.000 ETH) - вы сможете вернуть деньги после окончания ICO (когда закончится 4 неделя). До окончания ICO все собранные эфиры блокируются внутри контракта.

Но ведь собранные эфиры не блокируются внутри контракта, а сразу начисляются бенифициару, насколько я понял из исходников.

**Comment by hlogeon** [comment]
Строго говоря текстовые условия и смарт-контракт должны быть отображениями друг друга и не вступать в противоречия.

**Comment by prudnitskiy** [comment]
Нет, собранные эфиры перечисляются в Vault, и находятся там до окончания crowdsale. Если на момент finalize цель не достигнута (собранная сумма меньше goal) - перечисления бенифициару не происходит. Но мы проверим условия еще раз, спасибо.

**Comment by hlogeon** [comment]
Но ваш Vault и адрес контракта самого sale - разные вещи, а в тексте говорится о том, что вы блокируете все средства внутри контракта. Этого на самом деле не происходит
